### ISSUE TITLE
Learning to Respond with Your Favorite Stickers: A Framework of Unifying Multi-Modality and User Preference in Multi-Turn Dialog (1)

### ISSUE THREAD

**Issue by hs-sugi** [One-Day-One-Paper/One-Day-One-Paper#29]
# Learning to Respond with Your Favorite Stickers: A Framework of Unifying Multi-Modality and User Preference in Multi-Turn Dialog
- Shen Gao, Xiuying Chen, Li Liu, Dongyan Zhao, Rui Yan
- ACM Transactions on Information Systems, Vol. 39, No. 2, Article 12. Publicaton data: February 2021., pp.1-33.
- https://arxiv.org/pdf/2011.03322.pdf

### Abstract
- 사람마다 가장 좋아하는 이모티콘(스티커)를 통해서 추천할 수 있도록 학습시킴; 텔레그램 이모티콘을 통한 학습;
- 이모티콘의 스타일은 다르지만 이모티콘의 의미가 가까운 정도를 파악하여 적절한 이모티콘 추천
- 즉, 대화의 맥락을 알아듣고 이에 대해 알맞은 이모티콘을 여러 스타일로 추천이 됨.
- Sticker encoder, Utterance encoder, User preference modeling, Interaction network(이모티콘, 대화 간 매칭 수행)

![image](https://user-images.githubusercontent.com/40443049/148680048-9aaa3b95-4b12-4547-a684-df1b7eae7d3f.png)
![image](https://user-images.githubusercontent.com/40443049/148680058-e6bf6cd7-023a-47f8-9c8d-8c367e45c2cc.png)
- Overview 
![image](https://user-images.githubusercontent.com/40443049/148680085-a5861060-7d15-421f-8621-6d2dbefbfc2c.png)

- Sticker Encoder
    - ![image](https://user-images.githubusercontent.com/40443049/148680417-667a3286-e0f5-453f-9943-1b22be0e6327.png)
    > We utilize one of these models, i.e.,the Inception-v3 [63] model rather than plain CNN to encode sticker image; use the cross-entropy loss;

- Utterance Encoder
    - ![image](https://user-images.githubusercontent.com/40443049/148682305-a3db00c3-5ffc-43ab-863b-09b506d12fef.png)

- User Preference Modeling Module
    - ![image](https://user-images.githubusercontent.com/40443049/148682394-206d9c50-183d-4587-bf0c-95c401492382.png)
    - ![image](https://user-images.githubusercontent.com/40443049/148682408-d079c79c-1f9b-4f55-845e-5a08f4f0a85b.png)

- Deep Interaction Network
    - ![image](https://user-images.githubusercontent.com/40443049/148682346-672521c5-cc43-475c-82de-e12fc052a01e.png)
    - ![image](https://user-images.githubusercontent.com/40443049/148682353-6e7aa309-69f4-43b0-9108-464aca51030e.png)
    - ![image](https://user-images.githubusercontent.com/40443049/148682362-c0399cbb-d275-41c8-a595-1d473553e50c.png)
    - ![image](https://user-images.githubusercontent.com/40443049/148682371-3a8859a6-b6b2-4877-ba47-7d31e9168db0.png)
    - ![image](https://user-images.githubusercontent.com/40443049/148682379-17760294-0ab3-45fe-af95-b9116224a465.png)

- Fusion Network
    - matching score of the candidate sticker
    - ![image](https://user-images.githubusercontent.com/40443049/148682422-8a40b0c5-a5c4-4375-915f-b325be84fae4.png)

- Learning
    - ![image](https://user-images.githubusercontent.com/40443049/148682442-c7ecb117-dc48-4294-8410-513130d58626.png)
    - The gradient descent method is employed to update all the parameters in our model to minimize this loss function.

---- 
이해 못하고 후루룩 흐름만 잡았네요...
